tools/vscode-debugging/conf/launch.json

```diff
@@ -0,0 +1,49 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "interactive",
+            "type": "go",
+            "request": "launch",
+            "mode": "auto",
+            "preLaunchTask": "pre-debug",
+            "program": "${workspaceFolder}/cmd/launcher",
+            "args": [
+                "interactive",
+                "--osquery_flag=allow-unsafe",
+                "--osquery_flag=verbose",
+                // add atc config to the path below to load atc config
+                "--osquery_flag=config_path=${workspaceFolder}/debug/kolide-atc-config.json",
+            ],
+            "console": "integratedTerminal",
+            "asRoot": true
+        },
+        {
+            "name": "k2",
+            "type": "go",
+            "request": "launch",
+            "mode": "auto",
+            "preLaunchTask": "pre-debug",
```

Comment:
pre-debug is a shell script. What do you think about having it detect whether the enroll secret and cert are present and error out? (I don't know if it can produce a clear error that bubbles through)

Reply:
It's a little tricky because only need that stuff when running k2-localhost. It's not required for interactive. So then we have to have a condition in there and trying to pass the condition from the `launch.json` to the `task.json` (that runs before the debugger starts) is surprisingly difficult and requires many files and texts. Or I just haven't figured out the easy way. 

Also, the error isn't really in your face, you have to click the task's terminal tab to see it's output.

I would argue that launcher it self should emit a meaningful message if missing these files.

Reply:
I was actually able to drop the `pre-debug.sh` after realizing I could just call `make deps` in the task and launcher handles making it's own dirs

Reply:
> I would argue that launcher it self should emit a meaningful message if missing these files.

I could believe that. We can improve as we need them.